.github/workflows/checks.yml

```diff
@@ -6,14 +6,15 @@ concurrency:
 
 on:
   pull_request:
-  merge_group:
```

Comment:
This PR won't merge as GHA is not triggered anymore on the relevant commit in the merge queue: https://github.com/input-output-hk/ouroboros-consensus/commit/2e65ccb9c5fb5ac569d734091bc6af5dbdba8b67

Reply:
Same for the other workflows.